.github/boring-cyborg.yml

```diff
@@ -130,7 +130,7 @@ labelPRBasedOnFilePath:
     - airflow/bin/cli.py
     - airflow/cli/**/*.py
     - tests/cli/**/*.py
-    - docs/cli-ref.rst
+    - docs/cli-and-env-variables-ref.rst
```

Comment:
Can you add a new line to redirects.txt?
https://github.com/apache/airflow/blob/master/docs/redirects.txt

Reply:
Sounds good @mik-laj. I just added this.